test/yum/data_test.exs

```diff
@@ -185,6 +185,9 @@ defmodule Yum.DataTest do
     defp tree_counter({ _, tree }, n), do: tree_counter(tree, n + 1)
     defp tree_counter(tree, n), do: Enum.reduce(tree, n, &tree_counter/2)
 
+    defp list_counter(list, n \\ 0)
+    defp list_counter(list, n), do: Enum.reduce(list, n, fn _, n -> n+1 end)
```

Comment:
Don't actually need this, as a map implements the enumerable protocol. So what you've done here is actually equivalent to doing [`Enum.count/1`](https://hexdocs.pm/elixir/Enum.html#count/1) 😛